DigitalExcellence/dex-frontend
# Implement categories

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Create a project and add a category
2. Create a project without a category
3. Go to the overview page and make sure both are displayed correctly (detail page too)
4. Edit the project with a category, change to a different category and check step 3.
5. Delete the category from the project and check step 3.

## Link to issue

Closes: #475 
Closes: #493 
Closes: #351
